public/stylesheets/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  padding: 50px;
+  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+}
+
+a {
+  color: #00B7FF;
+}
+
+h4 {
+  display: inline;
+  margin-right: 1%;
+}
+
+form {
```

Comment:
I would avoid styling node names as much as possible. Use long, clear and unique classnames `.signup-form` etc